UnityProject/Assets/Scripts/Managers/GameManager.cs

```diff
@@ -376,10 +376,10 @@ private void UpdateMe()
 		{
 			if (NetworkTime.time >= CountdownEndTime)
 			{
-				if (PlayerList.Instance.ReadyPlayers.Count >= MinReadyPlayersForCountdown)
-				{
+				// if (PlayerList.Instance.ReadyPlayers.Count >= MinReadyPlayersForCountdown)
+				// {
 					StartRound();
-				}
+				// }
```

Comment:
Was this breaking something?

Reply:
oh yeah, forgot to uncomment, 
the issue is if you tab out and do something else while unity is loading, sometimes you can miss the end of the round start timer, and pops the joining up window but because no one is ready, It says the round hasn't started but the join window has appeared so you can't Ready up